app/src/utils/MadLibs.ts

```diff
@@ -0,0 +1,48 @@
+import STATE_FIPS_MAP from "./Fips";
+import { MadLib } from "./DatasetTypes";
+
+const MADLIB_LIST: MadLib[] = [
+  {
+    id: "1",
```

Comment:
Oh yeah I guess I didn't end up using them in the end! This does kinda make the MadLib type weirdly just have one property... we could refactor it so we just use the PhraseSegment lists directly but I'm kinda inclined to just leave it in case future properties we want to attach to the MadLib arise. I don't think refactoring it now vs later really changes the complexity much

Reply:
I agree with your reasoning, I like keeping the phrase segment as a property